web/themes/custom/instagram/css/all/footer.css

```diff
@@ -0,0 +1,72 @@
+.pC2e0{
```

Comment:
ignore generated Css files because they will be generated on local by executing gulp tasks.